docs/plugins/v0/legacy.rst

```diff
@@ -0,0 +1 @@
+.. warning:: The v0 plugin API is no longer the recommended way of developing plugins for Tutor, starting from Tutor v13.2.0. See our new :ref:`plugin creation tutorial <plugin_development_tutorial>` to learn more about the new v1 plugin API.
```

Comment:
I notice that the entrypoint for V1 plugins is currently `v1alpha`. I agree that it's reasonable to include "alpha" given the newness of the API. Should we hold off on dissuading usage of the V0 API until the V1 API isn't alpha anymore?

Reply:
... and also inject the word "new" between "developing" and "plugins".

That way it emphasizes that if you're starting a plugin from scratch, you might want to go with v1, but updates to existing plugins that use v0 are still worthwhile.

In contrast, I'd suggest removing the word "new" that precedes "v1 plugin API". It adds no additional value here, and new things become standard and then old quickly, at which point these little nits don't tend to get fixed down the road. That kind of word rot is easily averted by just dropping "new" here.